NAS-101423 doc the msg about available memory for VM (#1298)
freenas/freenas-docs: userguide/virtualmachines.rst

   |          |                    |                | number of CPUs.                                                                               |
   |          |                    |                |                                                                                               |
   +----------+--------------------+----------------+-----------------------------------------------------------------------------------------------+
   | 2        | Memory Size (MiB)  | integer        | Set the amount of RAM in megabytes                                                            |

>    |          |                    |                | for the VM. The available host system memory is displayed above the field.                    |

Reviewer: "available" suggests it is available right now.  Instead, it is just the amount installed.
```suggestion
   |          |                    |                | for the VM. The host system memory is displayed above the field.                              |
```

   |          |                    |                |                                                                                               |
   +----------+--------------------+----------------+-----------------------------------------------------------------------------------------------+

>    | 2        | Memory Size (MiB)  | integer        | Allocate the amount of RAM in                                                                 |

Reviewer: ```suggestion
    | 2        | Memory Size (MiB)  | integer        | Set the amount of RAM in megabytes                                                                |
```

>    |          |                    |                | `mebibytes <https://simple.wikipedia.org/wiki/Mebibyte>`__                                    |

Reviewer: ```suggestion
```
